src/oop_ext/interface/_interface.py

```diff
@@ -747,7 +747,7 @@ def AssertImplementsFullChecking(
 
 
 # set of methods that might be declared in interfaces but should be not be required by implementations
-_INTERFACE_METHODS_TO_IGNORE = {"__init_subclass__"}
+_INTERFACE_METHODS_TO_IGNORE = {"__init_subclass__", "__class_getitem__"}
```

Comment:
Was this added to keep python 3.6 support?

Reply:
No, this is the actual fix and is independent from the Python version.